Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -320,8 +565,10 @@ static void HandleQuery()
   else if (!strncmp((const char*)(s_cmd_bfr), "qThreadExtraInfo", strlen("qThreadExtraInfo")))
     return SendReply("00");
   else if (!strncmp((const char*)(s_cmd_bfr), "qSupported", strlen("qSupported")))
-    return SendReply("swbreak+;hwbreak+");
-
+    return SendReply("swbreak+;hwbreak+;qXfer:features:read+");
+  else if (!strncmp((const char*)(s_cmd_bfr),
+                    "qXfer:features:read:target.xml:", strlen("qXfer:features:read:target.xml:")))
+    ProcessXFerCommand(target_xml);
```

Comment:
One one hand: the condition spans multiple lines, so our code style dictates you add curlies (both here and everywhere else).

On the other hand though: Thoughts on moving this to a helper method, where the string isn't duplicated? Getting something wrong on the `strlen` call is fairly easy and even the `strcmp` ones are probably better off using `strncmp` (or any appropriate equivalent) anyways.

(plus it might get the line length down enough so the curlies don't have to be there.)

Reply:
The thing is that I was in progress of adding pr #10321 which WILL need the curly anyway for other reasons (I realised they both are close enough that they should be the same pr, but 2 different commits)

I'll do the helper thing though, but just know that the pr isn't yet ready for review, I'll move it out of draft when it is